PreFreeSurfer/PreFreeSurferPipeline.sh

```diff
@@ -203,94 +203,113 @@ fi
+                                      not used
+  --SEPhaseNeg={<file path>, NONE}    For spin echo field map, path to volume with
+                                      a negative phase encoding direction (LR in
+                                      HCP data), set to "NONE" if not using Spin
+                                      Echo Field Maps
+  --SEPhasePos={<file path>, NONE}    For spin echo field map, path to volume with
+                                      a positive phase encoding direction (RL in
+                                      HCP data), set to "NONE" if not using Spin
+                                      Echo Field Maps
+  --seechospacing=<seconds>           Effective Echo Spacing of Spin Echo Field Map,
+                                      (in seconds) or "NONE" if not used
+  --seunwarpdir={x,y,NONE}            Phase encoding direction (according to the *voxel* axes)
+             or={i,j,NONE}            of the spin echo field map. 
+                                      (Only applies when using a spin echo field map.)
+  --t1samplespacing=<seconds>         T1 image sample spacing, "NONE" if not used
+  --t2samplespacing=<seconds>         T2 image sample spacing, "NONE" if not used
+  --unwarpdir={x,y,z,x-,y-,z-}        Readout direction of the T1w and T2w images (according to the *voxel axes)
+           or={i,j,k,i-,j-,k-}        (Used with either a gradient echo field map 
+                                      or a spin echo field map)
+  --gdcoeffs=<file path>              File containing gradient distortion
+                                      coefficients, Set to "NONE" to turn off
+  --avgrdcmethod=<avgrdcmethod>       Averaging and readout distortion correction
+                                      method. See below for supported values.
+  [--custombrain=(NONE|MASK|CUSTOM)]  If PreFreeSurfer has been run before and you have created a custom
+                                      brain mask saved as "<path>/T1w/custom_acpc_dc_restore_mask.nii.gz", specify "MASK". 
+                                      If PreFreeSurfer has been run before and you have created custom structural images, e.g.: 
+                                      - "<path>/T1w/T1w_acpc_dc_restore_brain.nii.gz"
+                                      - "<path>/T1w/T1w_acpc_dc_restore.nii.gz"
+                                      - "<path>/T1w/T2w_acpc_dc_restore_brain.nii.gz"
+                                      - "<path>/T1w/T2w_acpc_dc_restore.nii.gz"
+                                      to be used when peforming MNI152 Atlas registration, specify "CUSTOM".
+                                      When "MASK" or "CUSTOM" is specified, all the steps until Atlas registration 
+                                      are skipped.
+                                      If the parameter is omitted or set to NONE (the default), 
+                                      standard image processing will take place.
+
+
+    "${NONE_METHOD_OPT}"
```

Comment:
This and below should be recombined with the `--avgrdcmethod` line.

Reply:
Done.